### PR TITLE
Automated Tests

### PR DESCRIPTION
**Pull request by AnashaSarker2001**
The first automated test was created. This test checks if the main page shows up in the application.

### :ballot_box_with_check: Definition of Done checklist
- [x] Meaningful title and valuable description in pull request
- [x] Code change tested manually if not covered by automated tests
- [x] Pull request linked to one or more issues/stories
- [x] Acceptance criteria of the linked stories satisfied
- [x] Code change comprehended by those who approved it
- [x] State of target branch maintained or improved after the code change

### CODE REVIEW

**Review by Gussamanel — APPROVED** [review]
Well done, the tests look fine.

File: LittleDrivers/app/src/androidTest/java/com/example/littledrivers/MainActivityTest.kt [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import com.example.littledrivers.MainActivity
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4ClassRunner::class)
+class MainActivityTest {
+    @Test
+    fun test_isActivity_inView() {
+        val activityScenario = ActivityScenario.launch(MainActivity::class.java)
+        onView(withId(R.id.main)).check(ViewAssertions.matches(ViewMatchers.isDisplayed()))
```
Comment (platisd) [inline LittleDrivers/app/src/androidTest/java/com/example/littledrivers/MainActivityTest.kt:19]: What is tested exactly here?

**Comment by platisd** [comment]
It should be noted that the newly added test does *not* run automatically on CI, which means that while the test exists it provides limited value since it is not run every time someone introduces a change.

**Comment by AnashaSarker2001** [comment]
Thank you for telling us, we are looking into CI so we can integrate it into GitHub.

**Pull request merged**